python/lsst/ip/diffim/dcrModel.py

```diff
@@ -549,23 +552,37 @@ def applyDcr(maskedImage, dcr, warpCtrl, bbox=None, useInverse=False):
         Shifts the entire image if None (Default).
     useInverse : `bool`, optional
         Use the reverse of ``dcr`` for the shift. Default: False
+    useMidpoint : `bool`, optional
+        Use the midpoint of each subfilter to calculate DCR. Default: True
```

Comment:
Can you clarify in the doctoring what the behavior is if this is False? I think it uses the start and endpoint of each filter instead, but saying that here would be helpful.

Reply:
I'm going to actually rename `useMidpoint` to `splitSubfilters`, with a default of False. That will make the documentation more clear.